scripts/package.json

```diff
@@ -49,7 +49,7 @@
   "bundlesize": [
     {
       "path": "../apps/test-bundle-button/dist/test-bundle-button.min.js",
-      "maxSize": "45.5 kB"
+      "maxSize": "45.7 kB"
```

Comment:
This is really bad. We can't just be adding more and more code and not thinking about ways to minimize the impact.